webapp/lib/model.dart

```diff
@@ -177,21 +167,64 @@ enum TagType {
   Important,
 }
 
-Map<String, Configuration> packageConfigurationData = {
-  'Change communications': new Configuration()
-    ..availableTags = tags
-    ..containsLastInTurnTags = {'denial': TagType.Normal , 'rumour': TagType.Normal}
-    ..hasNoneTags = {'escalate': TagType.Normal, 'STOP': TagType.Normal}
-    ..suggestedReplies = changeCommunicationsSuggestedReplies
-    ..addsTags = {'Organic conversation appreciation': TagType.Normal, 'Organic conversation hostility': TagType.Normal,
-      'RP Substance appreciation': TagType.Normal, 'RP Substance hostility': TagType.Normal},
-  'Urgent conversations': new Configuration()
-    ..availableTags = tags
-    ..hasAllTags = {'escalate': TagType.Important}
-    ..suggestedReplies = escalatesSuggestedReplies
-    ..addsTags = {'de-escalate': TagType.Normal, 'no-escalate': TagType.Normal},
-  'Open conversations': new Configuration(),
-};
+// Map<String, Configuration> packageConfigurationData = {
+//   'Change communications': new Configuration()
+//     ..availableTags = tags
+//     ..containsLastInTurnTags = {'denial': TagType.Normal , 'rumour': TagType.Normal}
+//     ..hasNoneTags = {'escalate': TagType.Normal, 'STOP': TagType.Normal}
+//     ..suggestedReplies = changeCommunicationsSuggestedReplies
+//     ..addsTags = {'Organic conversation appreciation': TagType.Normal, 'Organic conversation hostility': TagType.Normal,
+//       'RP Substance appreciation': TagType.Normal, 'RP Substance hostility': TagType.Normal},
+//   'Urgent conversations': new Configuration()
+//     ..availableTags = tags
+//     ..hasAllTags = {'escalate': TagType.Important}
+//     ..suggestedReplies = escalatesSuggestedReplies
+//     ..addsTags = {'de-escalate': TagType.Normal, 'no-escalate': TagType.Normal},
+//   'Open conversations': new Configuration(),
+// };
+
+List<Map> activePackages = [
```

Comment:
Let's make this a Map instead of a List, where the key is the id. It creates a bit of duplication here, but it will match how we handle the model in Nook and you won't need to have a `_findActivePackageById` method.

Reply:
Changed it into a `Map`